docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -53,4 +54,4 @@ See https://github.com/che-incubator/devworkspace-api/issues/4[corresponding iss
 * Renaming dockerimage component type. See  https://github.com/che-incubator/devworkspace-api/issues/8[corresponding issue].
 * Specify sources path for containers. See  https://github.com/che-incubator/devworkspace-api/issues/17[corresponding issue].
 * Specify size of volume for component. See https://github.com/che-incubator/devworkspace-api/issues/14[corresponding issue].
-* Containers endpoints (routes/ingresses). See https://github.com/che-incubator/devworkspace-api/issues/33[corresponding issue].
\ No newline at end of file
+* Containers endpoints (routes/ingresses). See https://github.com/che-incubator/devworkspace-api/issues/33[corresponding issue].
```

Comment:
Link change. See https://github.com/devfile/docs/pull/39#discussion_r578725087 for details.